.buildkite/pipeline.yml

```diff
@@ -54,6 +54,59 @@ steps:
     soft_fail:
       - exit_status: 3
 
+  - label: "oneAPI Julia {{matrix.version}}"
+    matrix:
+      setup:
+        version:
+          - "1.8"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "{{matrix.version}}"
+      - JuliaCI/julia-coverage#v1:
+          codecov: true
+    command: |
+      julia -e 'println("--- :julia: Instantiating project")
+                using Pkg
+                Pkg.develop(; path=pwd())
+                Pkg.develop(; name="oneAPI")' || exit 3
+
+      julia -e 'println("+++ :julia: Running tests")
+                using Pkg
+                Pkg.test("oneAPI"; coverage=true, test_args=["kernelabstractions"])'
+    agents:
+      queue: "juliaecosystem"
+      intel: "*"
+    timeout_in_minutes: 120
+    soft_fail:
+      - exit_status: 3
+
+  - label: "AMDGPU Julia {{matrix.version}}"
+    matrix:
+      setup:
+        version:
+          - "1.7"
+          - "1.9-"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "{{matrix.version}}"
+      - JuliaCI/julia-coverage#v1:
+          codecov: true
+    command: |
+      julia -e 'println("--- :julia: Instantiating project")
+                using Pkg
+                Pkg.develop(; path=pwd())
+                Pkg.develop(; name="AMDGPU")' || exit 3
+
+      julia -e 'println("+++ :julia: Running tests")
+                using Pkg
+                Pkg.test("AMDGPU"; coverage=true)'
```

Comment:
@pxl-th @jpsamaroo is there a way to run only a certain test?

Reply:
Not at the moment :/